### PR TITLE
BUGFIX: Avoid PHP exception in NamespaceDetectionTemplateProcessor

### PR DESCRIPTION
**Pull request by kdambekalns**
Related to neos/neos-development-collection#2479

### CODE REVIEW

File: Neos.Neos/Resources/Private/Templates/Backend/Module/Index.html [hunk 1]
```diff
@@ -28,10 +28,10 @@
 		</f:for>
 	</f:if>
 
-	<script type="text/javascript">
+	<script type="text/javascript"><![CDATA[
 		// TODO: Get rid of those global variables
 		{neos:backend.javascriptConfiguration()}
```
Comment (daniellienert) [inline Neos.Neos/Resources/Private/Templates/Backend/Module/Index.html]: Is this line still evaluated by fluid when wrapped in CDATA[] ?

Comment (bwaidelich) [reply Neos.Neos/Resources/Private/Templates/Backend/Module/Index.html]: no, I don't think it is!

File: Neos.Neos/Resources/Private/Templates/Backend/Module/Index.html [hunk 2]
```diff
@@ -73,13 +73,13 @@
 		<script src="{f:uri.resource(path: '2/js/bootstrap.min.js', package: 'Neos.Twitter.Bootstrap')}"></script>
 		<script src="{f:uri.resource(path: 'Library/fixedsticky/fixedsticky.js', package: 'Neos.Neos')}"></script>
 
-		<script type="text/javascript">
+		<script type="text/javascript"><![CDATA[
```
Comment (bwaidelich) [inline Neos.Neos/Resources/Private/Templates/Backend/Module/Index.html]: Maybe not important here, but for good measure CDATA blocks should be commented in javascript
```suggestion
		<script type="text/javascript">/*<![CDATA[*/
```

File: Neos.Neos/Resources/Private/Templates/Backend/Module/Index.html [hunk 3]
```diff
@@ -73,13 +73,13 @@
...
 			(function($) {
 				$(function() {
 					$('.neos-footer').fixedsticky();
 				});
 			})(jQuery);
-		</script>
+		]]></script>
```
Comment (bwaidelich) [inline Neos.Neos/Resources/Private/Templates/Backend/Module/Index.html]:
```suggestion
		/*]]>*/</script>
```

File: Neos.Neos/Resources/Private/Templates/Backend/Module/Index.html (hunk 1 above)
Comment (daniellienert) [reply Neos.Neos/Resources/Private/Templates/Backend/Module/Index.html]: So if this causes the problem, it needs to be fixed another way.

Comment (kdambekalns) [reply Neos.Neos/Resources/Private/Templates/Backend/Module/Index.html]: Right, I added this "after the fact" to have all JS wrapped for consistency. Not needed and breaks the inclusion (even though the module still loads.)

File: Neos.Neos/Resources/Private/Templates/Backend/Module/Index.html (hunk 2 above)
Comment (kdambekalns) [reply Neos.Neos/Resources/Private/Templates/Backend/Module/Index.html]: Not needed, the CDATA does not even arrive in the browser.

**Pull request merged**